modules/tls_openssl/openssl_config.c

```diff
@@ -770,7 +771,7 @@ int openssl_init_tls_dom(struct tls_domain *d, int init_flags)
 #endif
 		if (((void**)d->ctx)[i] == NULL) {
 			LM_ERR("cannot create ssl context for tls domain '%.*s'\n",
-				d->name.len, ZSW(d->name.s));
+				d->name.len, redact_pii(d->name.s));
```

Comment:
Is this the TLS domain name set in the configuration? If so, I think it does not need to be redacted.

Reply:
Same applies to the `tls_mgm` changes above I think?